integrationtests/as-lucene-directory/src/test/java/org/infinispan/test/integration/as/wildfly/model/Member.java

```diff
@@ -27,7 +37,30 @@
    @GeneratedValue
    private Long id;
 
-   @Field(index = Index.YES, store = Store.NO)
+   @Latitude
+   Double latitude;
+
+   @Longitude
+   Double longitude;
```

Comment:
Ugh, thanks @Sanne! I'll revert my last commit. I guess that's what I get for listening to @rhusar! :-)